assets/src/edit-story/components/library/mediaLibrary.js

```diff
@@ -16,12 +16,28 @@ import { useEffect } from '@wordpress/element';
 import UploadButton from '../uploadButton';
 import useLibrary from './useLibrary';
 
+const Container = styled.div`
+	display: grid;
+    grid-gap: 10px;
+    grid-template-columns: 1fr 1fr;
+`;
+
+const Column = styled.div`
+
```

Comment:
Is this still missing style? Otherwise we can remove the empty line.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3907/commits/d02b0b72525a12bbc64d5ee2412ec3ea958a0665